src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -0,0 +1,116 @@
+from collections.abc import Mapping
+from typing import Any
+
+from sentry.eventstore.models import Event
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.sentry_apps import CONTROL_TASK_OPTIONS, TASK_OPTIONS
+from sentry.tasks.sentry_apps import build_comment_webhook as old_build_comment_webhook
+from sentry.tasks.sentry_apps import clear_region_cache as old_clear_region_cache
+from sentry.tasks.sentry_apps import (
+    create_or_update_service_hooks_for_sentry_app as old_create_or_update_service_hooks_for_sentry_app,
+)
+from sentry.tasks.sentry_apps import installation_webhook as old_installation_webhook
+from sentry.tasks.sentry_apps import (
+    process_resource_change_bound as old_process_resource_change_bound,
+)
+from sentry.tasks.sentry_apps import retry_decorator
+from sentry.tasks.sentry_apps import send_alert_event as old_send_alert_event
+from sentry.tasks.sentry_apps import (
+    send_resource_change_webhook as old_send_resource_change_webhook,
+)
+from sentry.tasks.sentry_apps import workflow_notification as old_workflow_notification
+
+
+@instrumented_task(name="sentry.sentry_apps.tasks.sentry_apps.send_alert_event", **TASK_OPTIONS)
+@retry_decorator
+def send_alert_event(
+    event: Event,
+    rule: str,
+    sentry_app_id: int,
+    additional_payload_key: str | None = None,
+    additional_payload: Mapping[str, Any] | None = None,
+) -> None:
+    old_send_alert_event(
+        event=event,
+        rule=rule,
+        sentry_app_id=sentry_app_id,
+        additional_payload_key=additional_payload_key,
+        additional_payload=additional_payload,
+    )
+
+
+@instrumented_task(
+    "sentry.sentry_apps.tasks.sentry_apps.process_resource_change_bound", bind=True, **TASK_OPTIONS
+)
+@retry_decorator
+def process_resource_change_bound(self, action, sender, instance_id, *args, **kwargs):
```

Comment:
Oddly enough a lot of untyped functions in this file don't trigger mypy :thonk, spitballing, but maybe because their callers don't use types? The functions are not in classes? idk, but yes we should type them while we're at it